config/schema/format_strawberryfield.schema.yml

```diff
@@ -151,6 +167,9 @@ field.formatter.settings.strawberry_paged_formatter:
     iiif_group:
       type: boolean
       label: 'Whether multiple media sources should use a single IIIF viewer or not.'
+    json_key_source:
```

Comment:
This means you have to remove the old mediasource key. Not used anymore? Or?

Reply:
Same here. Please look at the code that writes/reads these configs. If a property exists in your local, staged, live, settings does not mean anything other than the fact we had no schema going on or some other archipelago-deployment glitch. YAMLs and Schemas are rules by code logic.